### PR TITLE
Updates to vis-testing 

### PR DESCRIPTION
**Pull request by scottqueen-bixal**
## Checklist

I have…

- [x] run the application locally (`make up`) and verified that my changes behave as expected.
- [x] run static behat test suite (`circleci build --job behat`) against my changes.
- [x] run the code sniffer (`circleci build --job code-sniffer`) against my changes.
- [x] run the code coverage tool (`circleci build --job code-coverage`) 
      against my changes
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.

## Summary of Changes

This pull request…

- updates our backstopjs paths array to include all current urls as of Friday, June 15, 2018
- includes a new make build-tools command for building our react tools

## Testing

To verify the changes proposed in this pull request…

**validate backstop testing**
1. update local with the latest DB, Files, Code, and Config
2. from the theme root, npm run vis-ref to create the reference images from stage
3. from the theme root, npm run vis-test to create the test images from your local and compare to stage
4. you should be able to pass all test currently
screen shot 2018-06-15 at 2 23 45 pm

**validate make command**
1. from the project root, run make build-tools
2. to confirm, navigate to each tool directory and ensure that 
- the node_modules have been installed
- the local css directory and css files have been compiled from sass
- the build directory has been created with the tool
- navigate to the the following urls and ensure that the tools render on your local env
/entitlements
/resources/locator-maps
/resources/weight-estimator
/resources/ppm-estimator

### CODE REVIEW

**Comment by scottqueen-bixal** [comment]
@sarnikbixal pinged you here as a reviewer in case you want to look at the bash script in the make file for building your sweet .mil tools?

**Comment by sarnikbixal** [comment]
Sorry Scott I was heads down all day freedom house, I’ll take a look at this first thing Monday. !RemindMe

Sent from my iPhone

On Jun 15, 2018, at 2:43 PM, Scott Queen <notifications@github.com<mailto:notifications@github.com>> wrote:


@scottqueen-bixal<https://github.com/scottqueen-bixal> requested your review on: Bixal/move.mil#188<https://github.com/Bixal/move.mil/pull/188> Updates to vis-testing .

File: Makefile [hunk 1]
```diff
@@ -20,10 +20,10 @@ prune:
 	@docker-compose down -v
 
 ps:
-	@docker ps --filter name='$(PROJECT_NAME)*'
+	@docker ps --filter name="$(PROJECT_NAME)*"
 
 shell:
-	docker exec -ti $(shell docker ps --filter name='$(PROJECT_NAME)_php' --format "{{ .ID }}") sh
+	docker exec -ti "$(shell docker ps --filter name="$(PROJECT_NAME)_php" --format "{{ .ID }}") sh"
```
Comment (ivonne-gov) [inline Makefile]: this command was working before, I don't know why you added the quotation marks but it is not working :( 
As it is now with the quotation mark, the command now it is confused and wrapping things different that you are expecting, for example, you are saying that this is one argument:
"$(shell docker ps --filter name="
and this is another one:
" --format "

Comment (scottqueen-bixal) [reply Makefile]: Hi @ivonne-gov I ran the code through a linter and that was the recommended update. I thought I had tested all the make commands, but maybe I missed this one.

It makes sense what you are explaining though.  Let me go in and update back to our previous state and push up the fix.

**Comment by scottqueen-bixal** [comment]
@ivonne-gov update per request on b548f99 
should execute as expected now, here I am demonstrating locally;

make-shell-fix

**Pull request merged**